app/Http/Controllers/ProjectCommentController.php

```diff
@@ -50,7 +50,13 @@ public function edit(ProjectComment $comment)
             return redirect('/')->withErrors('You are not authorized to do this action!');
         }
 
-        return view('project_comments.edit', compact('comment'));
+        $route = 'comments';
+        $object = 'message';
+        $routeToDelete = '/comments/'.$comment->id;
```

Comment:
In most cases I'm against the one liner version. It's too compact to read. There are too much information on one line and with a versioning like Git, having one idea per line is much better when doing diff.